inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -75,6 +77,58 @@ def from_text_file(
         depths = np.loadtxt(text_file_path)
         return cls(depths, x_range, y_range)
 
+    @classmethod
+    def from_random_depths(
+        cls,
```

Comment:
what is cls? is this self?

Reply:
Nop, it's the class. This is a [classmethod](https://docs.python.org/3/library/functions.html?highlight=classmethod#classmethod). The syntax follows commons Python code (inc. the Google style guide, see [this](https://google.github.io/styleguide/pyguide.html#3191-general-rules)).